### ISSUE TITLE
Fix missing breadcrumb parent

### ISSUE THREAD

**Issue by rcantin-w** [wellcomecollection/wellcomecollection.org#8771]
As flagged in Slack, some pages don't have a parent page, but the breadcrumb component still gets half-rendered, causing navigational issues (flagged in the 2021 a11y report)

[See thread in Slack](https://wellcome.slack.com/archives/C8X9YKM5X/p1666885544347049). 

![Image](https://user-images.githubusercontent.com/110461050/198341485-5dcfe075-eace-4e19-b11d-d9a672a3a490.png)

The decided fix was to ensure all pages had a parent (root being "Home").

**Comment by alexwlchan** [comment]
## For immediate consideration

Here are two things we could do pretty quickly:

* We've said that all pages should have a parent, the root being "Home", but we haven't actually done that.
* Press releases don't have any breadcrumb (e.g. https://wellcomecollection.org/pages/Y5n_fxEAAF9hr92I), but they are tagged consistently in Prismic as `press`. We could create a rule that says "if a page is tagged with 'press'", it gets a breadcrumb "Home / About us / Part of Media office", which scoops up significant chunk of our untagged content.

## For future consideration

I did a brief spike on Tuesday, looking at ways we could improve breadcrumb consistency. As Jenn says in the thread, in an ideal world we don't want this to be something that's managed manually by content editors – that gets unwieldy – but it is the system we currently have.

We have tools for analysing Prismic content in bulk, and we analyse the documents as a whole, not just as individual pages. If we look at the tagging/breadcrumbs for each page, we can start to see it as a tree, for example:

```
collections
	YDaP2BMAACUAT7DS (Databases)
	X_2eexEAACQAZLBi (Library membership)
	YA64vRMAACAAgRjZ (Accessing our collections )
	YCAVKxIAACIAV1oF (Library facilities)
	  	YBGdERMAAB8AjXKe (Copying and photography)
  		YBGcehMAAB4AjW_c (Computers, printing and wifi)
	  	YBGczxMAACAAjXFz (Book a study room)
	YBfeAhMAACEAqBTx (Collections)
	YBmCmhMAAB4Ar0G1 (Events for researchers)
	YLnuVRAAACMAftOt (The history and context of our collections)
	YFiLgBAAACoAKF7b (Collections offers and donations)
	YCUJ1hUAACcAUi5L (Inter-library loans)
	YE99nRAAACMAb7YE (What’s in the collections)
```

I only had a very brief look at this, just to establish the data is there to do it – and yes, it is.

We could use this to run validation rules against page breadcrumbs, for example:

* Does every page that's not a top-level item have a breadcrumb?
* Does every page have a site section?
* Are there any incomplete breadcrumbs? i.e.

    ```
    on page C: breadcrumb = A / B / C

    on page D: breadcrumb = C / D
    ```

This still puts the onus on somebody to tag the content manually, but building this visualisation tool properly would help us identify gaps in our breadcrumb strategy.

Here's the complete output for the site: [structure.txt](https://github.com/wellcomecollection/wellcomecollection.org/files/10235074/Untitled.txt)

**Comment by alexwlchan** [comment]
I'm going to add the "Home" prefix to all breadcrumbs.

**Comment by alexwlchan** [comment]
The "Home" prefix is done; I'm going to leave this ticket open and move it back into the backlog for the tree viewer idea, and to track discussion around the breadcrumb trail on press releases.
